### PR TITLE
Added windows compatibility to package.json `tests` scripts

### PR DESCRIPTION
**Pull request by nidin**
+Minor typo fixes

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "noImplicitAny": true,
```
Comment (winksaville) [inline tsconfig.base.json]: Why do we need to remove noImplicitAny, I like it because if I forget to add a type it screams at me.

Comment (nidin) [inline tsconfig.base.json]: its duplicate. there is already one bellow.

Comment (nidin) [inline tsconfig.base.json]: Just make an approve comment if its OK or change request.

Comment (winksaville) [inline tsconfig.base.json]: Whoops, got it :)

**Pull request merged**